### PR TITLE
DialNumber Text

### PR DESCRIPTION
**Pull request by MichaelCrimando**
This proposal adds a text string to `DialNumber` so that apps can be more descriptive or user friendly with it.

### CODE REVIEW

**Comment by MichaelCrimando** [comment]
Hey @theresalech this one is good to go

File: proposals/NNNN-DialNumber_Text.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    </function>
+```
+
+## Potential downsides
+
+Added complexity to the HMI
+
+## Impact on existing code
+
+It the additional message would be ignored on older headunits
```
Comment (theresalech) [inline proposals/NNNN-DialNumber_Text.md]: This section seems incomplete.  Can you please review and revise to include the impact this feature would have on the existing code?  Thanks!

File: proposals/NNNN-DialNumber_Text.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# DialNumber Text
+
+* Proposal: [NNNN](NNNN-DialNumberText.md)
```
Comment (theresalech) [inline proposals/NNNN-DialNumber_Text.md]:
```suggestion
* Proposal: [NNNN](NNNN-DialNumber_Text.md)

```

**Comment by MichaelCrimando** [comment]
@theresalech  thanks, done!

**Comment by MichaelCrimando** [comment]
@theresalech done

**Pull request merged**